src/ERC1155s.sol

```diff
@@ -274,13 +308,121 @@ abstract contract ERC1155s is ERC1155 {
     /// @dev Compute return string from baseURI set for this contract and unique vaultId
     function uri(
         uint256 superFormId
-    ) public view virtual override returns (string memory) {
+    ) public view virtual returns (string memory) {
         return
             string(abi.encodePacked(_baseURI(), Strings.toString(superFormId)));
     }
 
     /// @dev Used to construct return url
     function _baseURI() internal view virtual returns (string memory);
+
+    /*//////////////////////////////////////////////////////////////
+                              ERC165 LOGIC
+    //////////////////////////////////////////////////////////////*/
+
+    /// @dev Implementation copied from solmate/ERC1155
+    function supportsInterface(bytes4 interfaceId) public view virtual returns (bool) {
```

Comment:
I'm not sure if we need to check for all those ids, right?

Reply:
ERC1155 EIP doc specifies it:

_Smart contracts implementing the ERC-1155 standard MUST implement the ERC-165 supportsInterface function and MUST return the constant value true if 0xd9b67a26 is passed through the interfaceID argument._